README.md

```diff
@@ -4,23 +4,10 @@
 
 ## Use in your project
 Import dependencies versions
-```xml
-  <dependencyManagement>
-    <dependencies>
-      <dependency>
-        <groupId>io.knotx</groupId>
-        <artifactId>knotx-dependencies</artifactId>
-        <version>${knotx.version}</version>
-        <type>pom</type>
-        <scope>import</scope>
-      </dependency>
-    </dependencies>
-  </dependencyManagement>
+```kotlin
+implementation(platform("io.knotx:knotx-dependencies:${project.version}"))
```

Comment:
Do we still support Maven?

Reply:
I don't think we should, there will be two files to maintain.
What are your thoughts on this @tomaszmichalak ?